class/defaults.yml

```
@@ -7,10 +7,16 @@ parameters:
       roles: {}
       clusterRoles:
         cluster-admin: cluster-admin
+        # This is the OpenShift role that syncs namespaces after creation (e.g. add system-relevant labels/annotations from Project)
+        namespace-security-allocation-controller: system:openshift:controller:namespace-security-allocation-controller
       subjects: {}
 
     reservedNamespaces:
       kubernetes: ['default', 'kube-*']
       openshift: openshift-*
       projectsyn: syn-*
       appuio: appuio-*
+
+    generateDefaultRoleBinding:
```

```suggestion
    generateDefaultOrganizationRoleBinding:
```

Adding organization to the name would make the meaning clearer without reading documentation.

---

I understand the argument.

However it's not set in stone that this rolebinding binds to the organization. Currently, we haven't discussed Teams in detail yet, but the mid- to long-term plan is not that the organization gets Admin permissions, rather a Team.

I'm not sure what a good alternative name is without risking breaking changes.

---

Maybe `customer` would be more generic? WDYT?

---

`customer` is not a term we have defined in the current Appuio Cloud architecture, so I would avoid it since `customer` is not part of the domain.
https://github.com/appuio/appuio-io-docs/blob/main/docs/modules/ROOT/pages/references/glossary.adoc

maybe just `generateDefaultRoleBindingInNewNamespaces` or is that too long?

---

It's long but i prefer it over `generateDefaultRoleBinding `. 👍

---

Okay, let's go with that one for the time being